static/app/views/starfish/utils/constants.tsx

```diff
@@ -0,0 +1 @@
+export const STARFISH_CHART_INTERVAL_FIDELITY = 'metrics';
```

Comment:
Can we add a quick comment about what a fidelity of 'metrics' means?

Reply:
👍 added comment!